### PR TITLE
fix for additinal easyauth settings

### PR DESCRIPTION
**Pull request by watahani**
There has been a change to the default behavior of App Service Authentication, and by default, only tokens obtained using the appId of the app created for App Service Authentication are allowed. In Custom Claims Provider, since the API is called with an access token obtained by Azure Active Directory Authentication Extensions (client ID is 99045fe1-7639-4a75-9d4a-577b6ca3810f), it is necessary to allow this client on the App Service authentication side.

### CODE REVIEW

**Comment by Court72** [comment]
Hi @watahani,

In order for this PR to pass PR review, the blurred sections in the updated screenshots need to be obscured by using an opaque color block the same color as the background or by replacing the personal information with fictitious information. For more information, see [Sensitive content guidelines for content published to Learn](https://review.learn.microsoft.com/en-us/help/platform/repo-secrets-alerts-guidelines), [FAQs for documenting GUIDs and keys securely](https://review.learn.microsoft.com/en-us/help/platform/contribute-sensitive-identifiers), and [Approved sensitive identifiers](https://review.learn.microsoft.com/en-us/help/platform/reference-sensitive-identifier).

Can you make those updates?

**Comment by Court72** [comment]
@cilwerner

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by watahani** [comment]
updated screenshot to use approved identifiers for appid and tenant id

File: docs/identity-platform/custom-extension-tokenissuancestart-configuration.md [hunk 1]
```diff
@@ -390,6 +392,8 @@ Use the following steps to add Microsoft Entra as an identity provider to your A
 1. For the **Issuer URL**, enter the following URL `https://{domainName}.ciamlogin.com/{tenant_id}/v2.0`, where
     - `{domainName}` is the domain name of your external tenant, in the form `{domainName}.contoso.com`.
     - `{tenantId}` is the tenant ID of your external tenant.
+1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
```
Comment (cilwerner) [inline docs/identity-platform/custom-extension-tokenissuancestart-configuration.md:395]: Also line L390 - Select **External configuration** as the tenant for your application

File: docs/identity-platform/custom-extension-tokenissuancestart-configuration.md [hunk 2]
```diff
@@ -390,6 +392,8 @@ Use the following steps to add Microsoft Entra as an identity provider to your A
...
+1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter tenant id in GUID format.
```
Comment (cilwerner) [inline docs/identity-platform/custom-extension-tokenissuancestart-configuration.md:396]:
```suggestion
1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter your external tenant ID.
```

File: docs/identity-platform/custom-extension-tokenissuancestart-configuration.md [hunk 3]
```diff
@@ -373,6 +373,8 @@ Use the following steps to add Microsoft Entra as an identity provider to your A
 1. Select **Workforce** as the tenant type.
 1. Under **App registration** select **Pick an existing app registration in this directory** for the **App registration type**, and pick the *Azure Functions authentication events API* app registration you [previously created](#step-1-register-a-custom-authentication-extension) when registering the custom claims provider.
 1. Enter the following issuer URL, `https://login.microsoftonline.com/{tenantId}/v2.0`, where `{tenantId}` is the tenant ID of your workforce tenant.
+1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
+1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter tenant id in GUID format.
```
Comment (cilwerner) [inline docs/identity-platform/custom-extension-tokenissuancestart-configuration.md:377]:
```suggestion
1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter your workforce tenant ID.
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
A couple of minor changes here. Thanks for addressing this.

File: docs/identity-platform/custom-extension-tokenissuancestart-configuration.md (hunk 1 above)
Comment (watahani) [reply docs/identity-platform/custom-extension-tokenissuancestart-configuration.md:395]: fixed. also changed L391

**Comment by watahani** [comment]
@cilwerner Thank you for your review. I have made the necessary corrections.

**Review by cilwerner — APPROVED** [review]
LGTM, this can be signed off now.

**Comment by cilwerner** [comment]
#sign-off

File: docs/identity-platform/custom-extension-tokenissuancestart-configuration.md [hunk 4]
```diff
@@ -385,11 +387,13 @@ Use the following steps to add Microsoft Entra as an identity provider to your A
...
-1. Under **App registration**, enter the `client_id` of the *Azure Functions authentication events API* app registration you [previously created](#step-1-register-a-custom-authentication-extension) when registering the custom claims provider.
+1. Select **External configuration** as the tenant type.
+1. Under **App registration**, select **Provide the details of an existing app registration** for the **App registration type**, and enter the `client_id` of the *Azure Functions authentication events API* app registration you [previously created](#step-1-register-a-custom-authentication-extension) when registering the custom claims provider.
 1. For the **Issuer URL**, enter the following URL `https://{domainName}.ciamlogin.com/{tenant_id}/v2.0`, where
     - `{domainName}` is the domain name of your external tenant, in the form `{domainName}.contoso.com`.
     - `{tenantId}` is the tenant ID of your external tenant.
+1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
+1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter your external tenant ID.
```
Comment (ShannonLeavitt) [inline docs/identity-platform/custom-extension-tokenissuancestart-configuration.md:396]:
```suggestion
1. Under **Client application requirement**, select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
1. Under **Tenant requirement**, select **Allow requests from specific tenants** and enter your external tenant ID.
```

File: docs/identity-platform/custom-extension-tokenissuancestart-configuration.md [hunk 5]
```diff
@@ -373,6 +373,8 @@ Use the following steps to add Microsoft Entra as an identity provider to your A
 1. Select **Workforce** as the tenant type.
 1. Under **App registration** select **Pick an existing app registration in this directory** for the **App registration type**, and pick the *Azure Functions authentication events API* app registration you [previously created](#step-1-register-a-custom-authentication-extension) when registering the custom claims provider.
 1. Enter the following issuer URL, `https://login.microsoftonline.com/{tenantId}/v2.0`, where `{tenantId}` is the tenant ID of your workforce tenant.
+1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
+1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter your workforce tenant ID.
```
Comment (ShannonLeavitt) [inline docs/identity-platform/custom-extension-tokenissuancestart-configuration.md:377]:
```suggestion
1. Under **Client application requirement**, select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
1. Under **Tenant requirement**, select **Allow requests from specific tenants** and enter your workforce tenant ID.
```

**Pull request merged**